README.adoc

```diff
@@ -21,6 +21,25 @@ image:https://badges.gitter.im/eclipse/microprofile-concurrency.svg[link="https:
 [[microprofile-concurrency]]
 MicroProfile Concurrency
 ------------------------
+
+'''
+== How to contribute
```

Comment:
Thanks for the info! However, I think this detail would be better suited for a CONTRIBUTING.md document, which we could perhaps link to at the bottom of the readme.

With the currently proposed changes, the contributing guidelines takes up quite a lot of space and is at the top of the README.

Reply:
Agree with @aguibert.  We are supposed to have a CONTRIBUTING.md document, but it's not enforced by Eclipse.  We want the README to help with a quick understanding of the component -- not how to contribute.

Reply:
I just posted a general inquiry about this approach on the MP forum since this affects all of the components that have this README PR submitted....

Reply:
Ok cool. Making the changes as requested.